plugin.audio.ilovemusic/addon.xml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon 
+    id="plugin.audio.ilovemusic"
+    name="[COLOR white]I Love Music[/COLOR]"
+    version="2.0.2.1" 
```

Comment:
Your version number does not match the one from the changelog.txt and of the PR name?

Reply:
Yes, version 2.0.2 was the first version wich I uploaded.
After fixing the errors, witch was reported by TravisBuddy, I changed the version number to 2.0.2.1 .